src/internet_identity/internet_identity.did

```diff
@@ -653,7 +653,7 @@ type SetDefaultAccountError = variant {
     };
     NoSuchAccount : record {
         anchor_number : UserNumber;
-        account_number : opt AccountNumber;
+        origin : FrontendHostname;
```

Comment:
This means you need to generate the frontend types again, right?

Reply:
Yes, good catch